ansible/library/dual_tor_facts.py

```diff
@@ -43,7 +43,7 @@ def parse_loopback_ips(self):
             loopback_ips[dut]['ipv4'] = self.vm_config['DUT']['loopback']['ipv4'][dut_num]
             loopback_ips[dut]['ipv6'] = self.vm_config['DUT']['loopback']['ipv6'][dut_num] 
 
-            for loopback_num in range(1, 3): # Generate two additional loopback IPs, Loopback1 and Loopback2
+            for loopback_num in range(1, 4): # Generate two additional loopback IPs, Loopback1 and Loopback2
```

Comment:
Could you update the comment to include Loopback3?

Reply:
Sure, will update.